### ISSUE TITLE
v20 mass emailing module returns errors creating a new emailing, listing the emailings present, or trying to deactivate the module.

### ISSUE THREAD

**Issue by edupulpillo** [Dolibarr/dolibarr#31034]
### Bugs

**BUG1.Trying to access to the list of emailings is displayed the following error:**
```
This website or feature is currently temporarily not available or failed after a technical error.
This may be due to a maintenance operation. Current status of operation (2024-09-19T09:23:25Z) are on next line...
Dolibarr has detected a technical error. You can read log file or set option $dolibarr_main_prod to '0' in your config file to get more information.
```

**BUG2.Trying to create a new emailing, is returned the error message:** 
 `Unknown column 'messtype' in 'field list'`

**BUG3. Is not possible to deactivate the module mass emailings** in Setup/Modules-Aplications , and is returned the following error
![image](https://github.com/user-attachments/assets/93884c83-9e73-488d-baa8-20d83f0a7b2b)

### Dolibarr Version

20.0.0

### Environment PHP

8.1.29

### Environment Database

MySQL or MariaDB 10.6.19-MariaDB

### Steps to reproduce the behavior and expected behavior

Select in dolibarr the menu Tools/Emailing List

### Attached files

![image](https://github.com/user-attachments/assets/29ff04d5-2c73-4ef5-be3b-cdb7f16a06e9)

Then after click in the menu item "List" is displayed the following:
![image](https://github.com/user-attachments/assets/4d5d7104-9aed-4d33-a475-a4f68ea15aed)

Ths is the error message trying to create a new emailing 
![image](https://github.com/user-attachments/assets/28e37ca0-f1f6-4720-87b2-a4f7b40bc2d1)

**Comment by edupulpillo** [comment]
We did a backup, restore the instance to v19 and repeat the update.
Then the problem is not present, and we can not reproduce it anymore, So it seems was a problem temporary in our shared hosting.
We can close this issue as not really existing problem, with nothing needed to fix.

**Comment by edupulpillo** [comment]
Fixed in version 20.0.1
